content/showcase/_index.html

```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/sktapsall/CheckerBOT"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
```

Comment:
You haven't added this image as part of your PR, when you do make sure it's exactly 1000px x 750px.

Reply:
How do I add the image to /img/media/showcase

Reply:
Added image.